docs/extensions/accounts/releasing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Releasing"
+weight: 70
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/extensions/accounts
+geekdocFilePath: releasing.md
+---
+
+{{< toc >}}
+
```

Comment:
In general, I would add a *Requirements* section and mention that having Go installed is neccessary.

Reply:
Good idea, done!